pom.xml

```diff
@@ -191,6 +191,28 @@
 				    </execution>
 				</executions>		
 			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
```

Comment:
Why this?

Reply:
Oh. I thought I removed that one. I'll remove that and commit again